paddle/fluid/eager/auto_code_generator/generator/eager_gen.py

```diff
@@ -2225,6 +2241,16 @@ def GenerateNodeDefinition(
   {out_assign_str}}} else {{
   {indent}{autograd_api_out} api_output = paddle::experimental::{self.namespace}{self.grad_api_contents['invoke']};
   {out_assign_str}{indent}}}
+  """
+        elif is_composite_grad_api:
```

Comment:
Leave TODO here to indicate strategy which will be used later, such as using composite only when we don't have backward kernel

Reply:
This should be statically generated here

Reply:
done